docs/changelog-fragments/479.feature.rst

```diff
@@ -0,0 +1 @@
+Allowed processing of a specific configuration file for the connection -- by :user:`donnerhacke`
```

Comment:
I'd rephrase this through something "Implemented reading connection details from a config file"

Reply:
```suggestion
Implemented reading connection details from a config file -- by :user:`donnerhacke`.
```